src/status_im2/contexts/chat/composer/handlers.cljs

```diff
@@ -87,10 +87,11 @@
                                                     max-height)
           new-height   (min new-height max-height)]
       (reset! content-height content-size)
-      (when (utils/update-height? content-size height max-height maximized?)
+      (when (utils/update-height? content-size height max-height)
         (reanimated/animate height new-height)
+        (reanimated/set-shared-value last-height new-height)
         (reanimated/set-shared-value saved-height new-height))
-      (when (= new-height max-height)
+      (when (>= new-height max-height)
```

Comment:
`new-height` should never exceed `max-height` as above we are using `new-height   (min new-height max-height)`